iib/workers/tasks/utils.py

```diff
@@ -299,7 +299,7 @@ def set_registry_token(token, container_image):
 @dogpile_cache(
     dogpile_region=dogpile_cache_region, should_use_cache_fn=skopeo_inspect_should_use_cache
 )
-def skopeo_inspect(*args, return_json=True):
+def skopeo_inspect(*args, return_json=True, validate_mt=False):
```

Comment:
Would it be possible to check a list of args to define if there is `--raw` argument? This way you need to change only this function. WDYT?

Reply:
I think it is just simpler to have this boolean as return_json as we only need to check this in get_resolved_bundles and --raw option is only passed in this method and get_resolved_image.